.github/.kube-linter-config.yaml

```diff
@@ -1,13 +1,15 @@
 
 checks:
+  ignorePaths:
+    - "/home/runner/work/konflux-ci/konflux-ci/.kube-linter/out-dependencies-keycloak-deployment.yaml"
+    - "/home/runner/work/konflux-ci/konflux-ci/.kube-linter/out-dependencies-registry.yaml"
+    - "/home/runner/work/konflux-ci/konflux-ci/.kube-linter/out-dependencies-tekton-results.yaml"
```

Comment:
What's the reason for ignoring those?

Reply:
for all of them, Kubelinter fails because "readOnlyRootFilesystem" is not set to true. 
when setting it to be true, the ci fails because apparently they need to be able to write and not to read only.
@avi-biton might be able to elaborate

Reply:
Need to check if there's a way to ignore a specific issue on a specific file rather than ignoring the file completely due to a specific issue.

Reply:
See if this helps: https://docs.kubelinter.io/#/configuring-kubelinter?id=ignoring-violations-for-specific-cases
As far as I understand, you'd need to patch those files with an annotation to ignore that specific check.